bot.py

```diff
@@ -55,85 +57,58 @@ async def autodelete(ctx, *args):
             autodelete_functions.add(channel)
         await ctx.send("Added to `autodelete`:\n {}".format(mentioned_channels))
 
+
 @bot.listen()
 async def on_message(msg):
     if autodelete_functions.in_autodelete(msg.channel.id) and not msg.author.bot:
         await msg.delete(delay=10)
 
-         
+
 ###############################################################################
 ###############################################################################
-            
+
 # MATCH COMMAND
 
 @bot.command()
-async def match(ctx, *args):
-    send_channel = ctx.guild.get_channel(channelmap_functions.get_send_channel_id(ctx.channel.id))
-    
+async def match(ctx, team1: Union[Role,Member,str], team2: Union[Role,Member,str], *, dt=''):
```

Comment:
Could you make the variable name `dt` a bit more descriptive?

Reply:
If it means changing the syntax a bit more that's chill

Reply:
I previously called it `datetime` but that would clash if you needed the library. I've renamed it to `match_day` for now.

I did consider separate positional `date` and `time`, but this would break `-d -t` and mean that time requires a date (or vice versa if swapped). Since `!match ... Friday` and `!match ... 9pm` are equally valid I wasn't sure what to do.

I was also looking at the `parsedatetime` library, which would replace the parsing code and support most time strings (`Friday`, `9pm`, `9/2`, etc).